samples/join-a-team-using-qr-code/nodejs/Readme.md

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
Sample name should be start with bot- like **"bot-join-a-team-using-qr-code"**

Reply:
This is a scenario specific sample and does not come under bot section so did not add the bot naming convention